components/TaskCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledTaskContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  border: solid 1px #ff3566;
+  box-shadow: 4px 4px 4px 0px rgba(0, 0, 0, 0.1);
+
+  width: 80%;
+  padding: 6px 22px;
+  border-radius: 16px;
+`;
+
+const StyledTaskName = styled.p`
+  font-size: 22px;
+  height: 24px;
+  line-height: 24px;
+  text-align: center;
+`;
```

Comment:
`TaskCard` represents an individual item in a list of tasks, it should be structured more semantically using an unordered list (ul) for accessibility and proper HTML semantics. Therefore I suggest to use an `li` element for `StyledTaskContainer`, and the surrounding list should be in the `tasksPage`.